### PR TITLE
add sub mul div + common_value

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/core/regular/impl/add.hpp [hunk 1]
```diff
@@ -32,20 +29,20 @@ add_(EVE_SUPPORTS(cpu_),
 // N parameters
 //================================================================================================
 template<decorator D, value T0, value... Ts>
```
Comment (jfalcou) [inline include/eve/module/core/regular/impl/add.hpp:31]: I think the `<concept/value.hpp>` is still needed.

File: include/eve/module/core/regular/impl/mul.hpp [hunk 2]
```diff
@@ -7,13 +7,8 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/concept/compatible.hpp>
-#include <eve/concept/value.hpp>
```
Comment (jfalcou) [inline include/eve/module/core/regular/impl/mul.hpp:11]: same remark, still needed.

File: include/eve/traits/common_value.hpp [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  //! @addtogroup traits
+  //! @{
+  //!
+  //! @struct common_value_t
+  //!
+  //! @brief Computes a type that can represent all values in a list of types.
+  //!
+  //! **Required header:** `#include <eve/traits/common_value.hpp>`
+  //!
+  //! Similar to `std::common_type`.
```
Comment (jfalcou) [inline include/eve/traits/common_value.hpp:23]: Don't copy paste documentation, this is not what `common_value` does.

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/core/regular/impl/add.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/impl/add.hpp:31]: value is included by common_value

File: include/eve/traits/common_value.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/traits/common_value.hpp:23]: ok, I will remove it at the next push

**Review by jfalcou — APPROVED** [review]

**Pull request merged**